capstone/fabfile.py

```diff
@@ -400,7 +400,7 @@ def show_slow_queries():
     try:
         rows = cursor.fetchall()
         output = "*capstone slow query report*\n"
-    except:
+    except Exception:
         print(json.dumps({'text': 'Could not get slow queries'}))
```

Comment:
flake8 update is angry at naked except clauses

Reply:
Wait, why did flake8 complain to you and not to me?

Reply:
I accidentally updated flake8. It'll complain in the future.